src/main/java/lanat/parsing/errors/ParseStateErrorBase.java

```diff
@@ -121,11 +112,9 @@ private boolean isHandlerMethod(@NotNull MMethod<?> method) {
 			if (!this.isHandlerMethod(method)) continue;
 
 			try {
-				method.bindToObject(this)
-					.requireAccessible()
-					.invoke();
+				method.bindToObject(this).requireAccessible().invoke();
```

Comment:
Should this be split up into multiple lines?
```suggestion
				method.bindToObject(this)
						.requireAccessible()
						.invoke();
```